lib/tessel/provision.js

```diff
@@ -256,7 +287,7 @@ function AlreadyAuthenticatedError() {
 util.inherits(AlreadyAuthenticatedError, Error);
 
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
-
+module.exports.TESSEL_AUTH_KEY = filepath;
```

Comment:
@rwaldron I am unsure this is what @johnnyman727 likes to get. This is a public accessible global used as base for the running CLI. The instantiated objects also can be changed by the setter - this one has another usage.